crates/sncast/tests/e2e/script/call.rs

```diff
@@ -99,12 +99,10 @@ async fn test_call_invalid_calldata() {
     assert_stdout_contains(
         output,
         indoc! {r#"
-        ScriptCommandError::ProviderError(ProviderError::StarknetError(StarknetError::ContractError(ErrorData { msg: "Error at pc=0:1401:
-        An ASSERT_EQ instruction failed: 5:2 != 5:5.
-        " })))
-        ScriptCommandError::ProviderError(ProviderError::StarknetError(StarknetError::ContractError(ErrorData { msg: "Error at pc=0:1401:
-        An ASSERT_EQ instruction failed: 5:2 != 5:1.
-        " })))
+        [..]Compiling[..]
+        [..]Finished[..]
+        ScriptCommandError::ProviderError(ProviderError::StarknetError(StarknetError::ContractError(ErrorData { msg: "Execution failed. Failure reason: 0x496e70757420746f6f206c6f6e6720666f7220617267756d656e7473 ('Input too long for arguments')." })))
+        ScriptCommandError::ProviderError(ProviderError::StarknetError(StarknetError::ContractError(ErrorData { msg: "Execution failed. Failure reason: 0x4661696c656420746f20646573657269616c697a6520706172616d202332 ('Failed to deserialize param #2')." })))
```

Comment:
Is this right? The previous error seemed different than this one, so maybe just double check

Reply:
I think this should be ok, this seems to be indicating invalid calldata